tools/labs/qemu/Makefile

```diff
@@ -25,10 +25,12 @@ QEMU_OPTS = -kernel $(ZIMAGE) \
 	-drive file=disk1.img,if=virtio,format=raw \
 	-drive file=disk2.img,if=virtio,format=raw \
 	--append "root=/dev/vda loglevel=15 console=hvc0" \
-	--display $(QEMU_DISPLAY) -s -m 256
+	--display $(QEMU_DISPLAY) -s -m 256 \
+	# -enable-kvm -cpu host
 
```

Comment:
This is a good change. We should set it via a env variable like QEMU_DISPLAY for example. 
```
ENABLE_KVM = yes ./make boot 
```

then the Makefile should be updated with something like this
```
if ENABLE_KVM
QEMU_OPTS += -enable-kvm
endf
```

Reply:
Done. Does it look better now?